ecs_model_deployer/src/lib/lisa_model_stack.ts

```diff
@@ -38,13 +38,28 @@ export class LisaModelStack extends Stack {
             vpcId: props.vpcId
         });
 
+        let subnetSelection: SubnetSelection | undefined;
+
+        if (props.config.subnetIds && props.config.subnetIds.length > 0) {
+            const iSubnets: Map<number, ISubnet> = new Map();
+            props.config.subnetIds.forEach((subnet, index) => {
+                const importedSubnet = Subnet.fromSubnetId(this, index.toString(), subnet);
+                iSubnets.set(index, importedSubnet);
+            });
+
+            subnetSelection = {
+                subnets: Array.from(iSubnets!.values()),
+            };
```

Comment:
This could be needed for a reason I don't understand but couldn't this be simplified by something like below? Seems like going to a map is an unnecessary step. 

```
subnetSelection = {
  subnets: props.config.subnetIds.map((subnet, index) => Subnet.fromSubnetId(this, index.toString(), subnet))
};